packages/cozy-pouch-link/package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "cozy-client": "^15.4.0",
     "cozy-device-helper": "^1.7.3",
-    "minilog": "^3.1.0",
+    "minilog": "https://github.com/cozy/minilog.git#master",
```

Comment:
It's problematic to have a git repository here, it's always moving in the yarn.lock, can you publish on npm ?

Reply:
Good idea! Done ;).